bin/usmqe_alerts_logger.py

```diff
@@ -0,0 +1,140 @@
+        self.username = username
+
+    def __repr__(self):
+        return "TendrlAuth(token={})".format(self.__bearer_token)
+
+    def __call__(self, r):
+        """
+        Add Tendl Bearer Token into header of the request.
+        For full description, see requests documentation:
+        http://docs.python-requests.org/en/master/user/authentication/
+        """
+        headers = {
+            "Authorization": "Bearer {}".format(self.__bearer_token),
+            }
+        r.prepare_headers(headers)
+        return r
+
+
+def login(username, password, url, asserts_in=None):
+    """
+    Login Tendrl user.
+    Args:
+        username: name of user that is going logged in
+        password: password for username
+        asserts_in: assert values for this call and this method
+    Returns requests auth object (instance of TendrlAuth)
+    """
+    pattern = "login"
+    post_data = {"username": username, "password": password}
+    request = requests.post(
+        url + pattern,
+        data=json.dumps(post_data))
+    token = request.json().get("access_token")
+    auth = TendrlAuth(token, username)
+    return auth
+
+
+class TendrlApi(object):
+    """ Common methods for Tendrl REST API.
+    """
+
+    def __init__(self, auth=None, url=None):
+        """
+        Args:
+            auth: TendrlAuth object (defines bearer token header), when auth is
+               None, requests are send without athentication header
+        """
+        # requests auth object with so called tendrl bearer token
+        self._auth = auth
+        self.url = url
+
+    def alerts(self):
+        """ Alerts REST API
+        Name:        "alerts",
+        Method:      "GET",
+        Pattern:     "alerts",
+        """
+        response = requests.get(
+            self.url + "alerts",
+            auth=self._auth)
+        return response.json()
+
+
+class TendrlFormatter(logging.Formatter):
+    """
+    This class can be used to create special formating of tendrl logs.
+    """
+    def format(self, record):
+        # TODO(fbalak): update this function to alter log format
+        result = super(TendrlFormatter, self).format(record)
+        return result
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("user", type=str, nargs='?')
+    args = parser.parse_args()
+    config = ConfigParser.RawConfigParser()
+    config.read("/etc/usmqe_alerts_logger_users.ini")
+    password = config.get(args.user, "password")
+    ssl = True if config.get(args.user, "ssl") == "true" else False
+    if ssl:
+        url = "https://" + config.get(args.user, "url") + "/api/1.0/"
+    else:
+        url = "http://" + config.get(args.user, "url") + "/api/1.0/"
+    auth = login(args.user, password, url)
+    api = TendrlApi(auth=auth, url=url)
+    logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+    logger = logging.getLogger("usmqe_alerts_logger@{}".format(args.user))
```

Comment:
There is one problem with this, your new logger reports every log entry and then handles it to the parent logger, so that there are 2 log entries reported for every entry, see: https://docs.python.org/2/library/logging.html#logging.Logger.propagate

The easiest thing (not sure if canonically correct) would be to  disable this propagation by adding line:

```
logger.propagate = False
```